deploy/docker/scripts/supervisor_event_listener.py

```diff
@@ -0,0 +1,70 @@
+import sys, shutil, requests, time, os
+
+LOADING_TEMPLATE_PAGE = r'/opt/appsmith/templates/appsmith_starting.html'
+LOADING_PAGE_EDITOR = r'/opt/appsmith/editor/loading.html'
+BACKEND_HEALTH_ENDPOINT = "http://localhost/api/v1/health"
+LOADING_PAGE_TIMEOUT_IN_SEC = 45
+
+def write_stdout(s):
+    # only eventlistener protocol messages may be sent to stdout
+    sys.stdout.write(s)
+    sys.stdout.flush()
```

Comment:
Make your life easier. 🙂

*Note* that with this change, you don't need the `\n` characters when calling `write_stdout`. The `print` function will do it for you.

```suggestion
def write_stdout(*args):
    # only eventlistener protocol messages may be sent to stdout
    print(*args, flush=True)
```

Reply:
I tried running the script with the changes but the event stream did not work.
I think in some cases the output streams don't need the newline to communicate to the supervisor that it is ready.